<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProgramView</class>
 <widget class="QDialog" name="CreateProgramView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Créer un nouveau programme</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="mainLayout" stretch="1,2">
     <item>
      <widget class="QListWidget" name="projectTypesList">
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="optionsLayout">
       <item>
        <layout class="QHBoxLayout" name="programNameLayout">
         <item>
          <widget class="QLabel" name="programNameLabel">
           <property name="text">
            <string>Nom du programme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="programNameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="programLocationLayout">
         <item>
          <widget class="QLabel" name="programLocationLabel">
           <property name="text">
            <string>Emplacement du programme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="programLocationEdit">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseLocationButton">
           <property name="text">
            <string>Parcourir</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/icons/open</normaloff>:/icons/open</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="defaultLocationCheckBox">
         <property name="text">
          <string>Faire de ce dossier le dossier par défaut</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateProgramView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateProgramView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
